apps/src/music/views/Controls.jsx

```diff
@@ -63,6 +65,13 @@ const Controls = ({
     </div>
   );
 
+  const documentationLinkIconSection = renderIconButton(
+    'question-circle-o',
+    () => {
+      analyticsReporter.onButtonClicked('documentation-link');
+      window.open(documentationUrl, '_blank', 'noopener,noreferrer');
```

Comment:
@Hamms Does this seem alright?

Reply:
As a general rule, we should avoid using `window.open`; it breaks accessibility, often runs afoul of popup blockers, and is generally considered an antipattern. See https://developer.mozilla.org/en-US/docs/Web/API/Window/open#accessibility for more

I will also always advocate for us to avoid opening links in new windows or tabs in any context. See https://www.w3.org/TR/WCAG20-TECHS/G200.html, https://grrr.tech/posts/opening-links-in-new-tabs/, https://marco.org/2014/01/10/target-blank, https://uxdesign.cc/linking-to-a-new-tab-vs-same-tab-f88b495d2187, https://medium.com/the-metric/links-should-open-in-the-same-window-447da3ae59ba, https://meiert.com/en/blog/links-in-new-tabs/, for more.

Reply:
Agreed!  My understanding is that this is just temporary while we incubate this, and going forward we'll do better inline documentation for blocks.  I did want to make sure I was using the appropriate parameters which I found used elsewhere.

Reply:
Those are indeed the correct parameters to address the domain takeover issue which is also a problem with links opened in a new tab. Note, however, that even that protection is not absolute but is merely the best we can do. My recommendation is for even temporary measures like this one to use regular HTML links

Reply:
> My recommendation is for even temporary measures like this one to use regular HTML links

Oh, good point!  Have updated to that.